### PR TITLE
Comment reporting links

### PR DESCRIPTION
**Pull request by jongilbraith**
Adds a mailto link under each comment with the text "Report this comment".

When clicked you get an email with subject of ``Reported comment in article 'Comment TItle'``.

The prepopulated body is like so:

```
Reported comment: http://blog.moneyadviceservice.org/comment-title#comment-1

Please enter any further notes below:

```

### CODE REVIEW

File: app/helpers/article_helper.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 module ArticleHelper
 
+  REPORT_COMMENT_EMAIL_ADDRESS = 'blog@moneyadviceservice.org.uk'
```
Comment (baphled) [inline app/helpers/article_helper.rb]: Really hoping we can get away with not using a constant ;D

Comment (jongilbraith) [inline app/helpers/article_helper.rb]: If people don't like constants, I shall not protest.

What would you prefer?

File: app/helpers/article_helper.rb [hunk 2]
```diff
@@ -11,4 +13,21 @@ def display_comment_timestamp(comment)
...
+  def report_comment_link(comment)
+    mail_to REPORT_COMMENT_EMAIL_ADDRESS,
+            t('.report_comment'),
+            subject: "Reported comment in article '#{comment.article.title}'",
+            body: report_comment_email_template(comment)
+  end
+
+  private
+
+  def report_comment_email_template(comment)
```
Comment (baphled) [inline app/helpers/article_helper.rb]: It'd be nice if we used a view template here :)

Comment (jongilbraith) [inline app/helpers/article_helper.rb]: I dunno, for just three lines?

With the crimes against programming that already exist in this codebase, it doesn't feel like that big a deal.

File: app/helpers/article_helper.rb (hunk 1 above)
Comment (baphled) [inline app/helpers/article_helper.rb]: Dunno, poss in each of the env files possibly.

File: app/helpers/article_helper.rb (hunk 2 above)
Comment (baphled) [inline app/helpers/article_helper.rb]: haha, broken windows ;D

File: app/helpers/article_helper.rb (hunk 1 above)
Comment (jongilbraith) [inline app/helpers/article_helper.rb]: Works for me, shall rework...

**Pull request merged**